lib/utils/validators.js

```diff
@@ -0,0 +1,42 @@
+class Validator {
+  constructor(test, errorMessage) {
+    this.test = test || false;
+    this.errorMessage = errorMessage || '';
+  }
+
+  _validate(input) {
+    if (!this.test) {
+      return false;
+    }
+    return this.test.test(input);
+  }
+
+  isValid(input) {
+    return this._validate(input);
+  }
+}
+
+
+const none = new Validator(false);
+const email = new Validator(/.+@.+\./, 'Not a valid email address');
```

Comment:
I would suggest that we just individually export all these validators.  I would also suggest that we name them like classes. So:
<pre>
export class Validator {
  ...
}
export EmailValidator = new Validator(...);  
export ZipCodeValidator = new Validator(...);  
export LowercaseStringValidator = new Validator(...);
export NumberValidator = new Validator(...);
</pre>

etc.